apps/web/src/components/Menu/index.tsx

```diff
@@ -56,7 +57,7 @@ const Menu = (props) => {
         rightSide={
           <>
             <GlobalSettings mode={SettingsMode.GLOBAL} />
-            <Notifications />
+            {props.isUserIpWhitelisted && <Notifications />}
```

Comment:
We can read the flag from the `ServerContext` here

Reply:
it would be more intuitive if we create a hook called `useNotificationEnabled`, which should read from both ServerContext and global settings. The global settings need to have three different states, which are initial/on/off. Because once the user turns it off manually we should keep it that way

Reply:
therefore the global settings could be either `undefined`, `true` or `false` to represent different situations

Reply:
<img width="377" alt="image" src="https://github.com/pancakeswap/pancake-frontend/assets/116779127/e269c5e1-e439-459d-9c4f-6f52eb1d8792">

The switch should be added here